java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -282,6 +282,27 @@ CompletableFuture<Double> zaddIncr(
      */
     CompletableFuture<Map<String, Double>> zpopmin(String key, long count);
 
+    /**
+     * Removes and returns up to <code>count</code> members with the lowest scores from the sorted set
+     * stored at the specified <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/zpopmin/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param count Specifies the quantity of members to pop.<br>
+     *     If <code>count</code> is higher than the sorted set's cardinality, returns all members and
+     *     their scores, ordered from lowest to highest.
+     * @return A map of the removed members and their scores, ordered from the one with the lowest
+     *     score to the one with the highest.<br>
+     *     If <code>key</code> doesn't exist, it will be treated as an empty sorted set and the
+     *     command returns an empty <code>Map</code>.
+     * @example
+     *     <pre>{@code
+     * Map<GlideString, Double> payload = client.zpopmax(gs("mySortedSet"), 2).get();
+     * assert payload.equals(Map.of(gs('member3'), 7.5, gs('member2'), 8.0)); // Indicates that gs('member3') with a score of 7.5 and gs('member2') with a score of 8.0 have been removed from the sorted set.
```

Comment:
nit: but lets be consist in strings either use " or '

Reply:
Changed in new commit to "